sdks/python/apache_beam/runners/direct/clock.py

```diff
@@ -15,36 +15,37 @@
 # limitations under the License.
 #
 
-"""Clock implementations for real time processing and testing."""
+"""Clock implementations for real time processing and testing.
 
+For internal use only. No backwards compatibility guarantees.
+"""
 from __future__ import absolute_import
 
 import time
 
 
 class Clock(object):
-  """For internal use only; no backwards-compatibility guarantees."""
-
   def time(self):
     """Returns the number of milliseconds since epoch."""
```

Comment:
<!--new_thread; commit:c1b215245684110f60ed943b720e17c0bdd5a8bc; resolved:0-->
Please fix comment to indicate seconds.